webClient/src/app/shared/model/editor-project.ts

```diff
@@ -26,18 +26,16 @@ export interface ProjectStructure {
 }
 
 export interface DatasetAttributes {
-  csiEntryType: string,
-  name: string,
-  dsorg?: DatasetOrganization,
-  members?: Array<any>,
-  recfm?: RecordFormat,
-  volser?: string
+  volser: string;
+  recfm: RecordFormat;
+  dsorg: DatasetOrganization;
```

Comment:
Why are we removing 'name' 'csiEntryType' 'members'  ?

Reply:
Removed that since they were not used. But still add that because we might need in future.